### PR TITLE
Merge profile pages

### PR DESCRIPTION
**Pull request by JAFAJ0**
### What this PR does (required):
- This PR combine and simplify the complex profile page which is because of merge and separate tasks
THIS PR SHOULD BE checked AFTER PROFILE PHOTO PR#66 IS APPROVED AND MERGED
### Screenshots / Videos (front-end only):
- 
ep

### Any information needed to test this feature (required):
- 

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: client/src/pages/ProfileSkeleton/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const { initSocket } = useSocket();
+  const history = useHistory();
+
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
+    history.push('/login');
+    // loading for a split seconds until history.push works
```
Comment (ezeilo-su) [inline client/src/pages/ProfileSkeleton/Profile.tsx:25]: Let's remove the comment

File: server/controllers/profilephoto.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        const pathname = profile.photoUrl.substring(48);
+        const deleteParams = {
+            Bucket: process.env.AWS_BUCKET,
+            Key: pathname,
+        }
+        await s3.deleteObject(deleteParams, (err, data) => {
+            if (err) {
+                res.send({ err: 'Error occured while trying to delete previous file in S3 bucket' });
+            }
+            console.log(data)
```
Comment (ezeilo-su) [inline server/controllers/profilephoto.js:32]: Please remove comment

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
A lot of progress here. Please can you link your ticket to this PR? Follow this [guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). Also, resolve the merge conflict before another review.

File: client/src/components/Bookings/BookingsSideBanner/BookingsSideBanner.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export default function BookingsSideBanner({ requests }: Props): JSX.Element {
       <Grid item>
         <NextBooking request={nextBooking} />
       </Grid>
-      <Grid item>{requests.length > 1 ? <BookingsPaper requests={requests} /> : ''}</Grid>
+      <Grid item>{requests.length > 1 && <BookingsPaper requests={requests} />}</Grid>
```
[the remark this answers is not in the document]
Comment (JAFAJ0) [reply client/src/components/Bookings/BookingsSideBanner/BookingsSideBanner.tsx]: Not my part of code and come in by merge issue

**Pull request merged**